docs/tools/fbsql/fbsql-loaders.md

```diff
@@ -0,0 +1,53 @@
+---
+title: fbsql-loaders
+layout: default
+parent: fbsql
+grand_parent: Tools
+nav_order: 2
+---
+
+# FBSQL Loaders
+
+`fbsql` is a CLI tool used to interact with FeatureBase using SQL. It can also be run in non-interactive mode to load data from supported data sources. These "loaders" connect to a data source, build batches of SQL `BULK INSERT` statements with the data returned from the source, and send those `BULK INSERT` requests to FeatureBase.
+
+Below are the currently supported data sources (with links to the source specific documentation):
+
+- [Kafka](/docs/tools/fbsql/fbsql-loaders-kafka):           use the `--loader-kafka` flag with the `fbsql` binary to run the kafka loader.
+- [PostgreSQL](/docs/tools/fbsql/fbsql-loaders-postgresql): use the `--loader-postgres` flag with the `fbsql` binary to run the postgres loader.
+- [Impala](/docs/tools/fbsql/fbsql-loaders-impala):         use the `--loader-impala` flag with the `fbsql` binary to run the impala loader.
+
```

Comment:
You need to insert a "Before you begin" heading because you can't use FBSQL without some dependencies.

For example:

```
## Before you begin
* [Learn about fbsql](/docs/tools/fbsql/fbsql-home/)
* [Learn how to install fbsql](/docs/tools/fbsql/fbsql-install)
* [Learn about Featurebase supported SQL](/docs/sql-guide/sql-guide-home)
```